portfolio/src/main/webapp/contact.html

```diff
@@ -0,0 +1,149 @@
+
+    <!--  Scripts from materalizecss.com-->
+    <script src='https://code.jquery.com/jquery-2.1.1.min.js'></script>
+    <script src='js/materialize.min.js'></script>
+    <script src='js/init.js'></script>
+
+
+    <link rel='stylesheet' href='css/style.css'>
+
+  </head>
+  <body>
+    <!-- Navigation Bar -->
+    <nav class='white'>
+      <div class='nav-wrapper container'>
+        <a id='logo-container' href='index.html' class='brand-logo'>Sara Missak</a>
+        <ul class='right hide-on-med-and-down'>
+          <li><a href='index.html'>About Me</a></li>
+          <li><a href='projects.html'>My Projects</a></li>
+          <li><a class='active'  href='contact.html'>Contact Me</a></li>
+        </ul>
+
+        <!-- For mobile use, edits -->
+        <ul id="nav-mobile" class="sidenav">
+          <li><a class='active' href='index.html'>About Me</a></li>
+          <li><a href='projects.html'>My Projects</a></li>
+          <li><a href='contact.html'>Contact Me</a></li>
+        </ul>
+        <a href="#" data-target="nav-mobile" class="sidenav-trigger"><i class="material-icons">menu</i></a>
+      </div>
+   	</nav>
+
+    <!-- First container with title -->
+	<div id='index-banner' class='parallax-container'>
+      <div class='section no-pad-bot'>
+        <div class='container'>
+          <br><br>
+          <h1 class='header center teal-text accent-3' id='title'>Contact me!</h1>
+          <br><br>
+        </div>
+      </div>
+      <!-- back ground image for first container -->
+      <div class='parallax'><img src='images/background3.JPG' alt='Unsplashed background img 1'></div>
+    </div>
+
+    <!-- Third (White) container -->
+    <div class='container'>
+      <div class='section'>
+        <div class='row'>
+          <div class='col s12 center'>
+        	<h3><i class='mdi-content-send brown-text'></i></h3>
+	        <h4>Send me a message!</h4>
+            <form class="col s12" onsubmit="return validate()" action="mailto:sarammissak@gmail.com" method="post" enctype="text/plain">
+              <div class="row">
+                <div class="input-field col s6">
+                  <i class="material-icons prefix">account_circle</i>
+                  <input id="firstName" type="text" class="validate" name="First Name" required>
+                  <label for="firstName">First Name</label>
+                </div>
+                <div class="input-field col s6">
+                  <input id="lastName" type="text" class="validate" name="Last Name" required>
+                  <label for="lastName">Last Name</label>
+                </div>
+              </div>
+  
+              <div class="row">
+                <div class="input-field col s12">
+                  <i class="material-icons prefix">phone</i>
+                  <input id="telephone" type="tel" class="validate" maxlength="10" name="Phone Number" required>
```

Comment:
Fixed length of 10 does not allow for country code. Also it prohibits users from copy-pasting phone numbers which might have separators in it. Our goal is to have users enter something valid without hampering their experience of the page.

An option here could be use a javascript based validator which validates the number on lost focus.